cspell.yaml

```diff
@@ -33,6 +33,26 @@ import:
 - "@cspell/dict-shell/cspell-ext.json"
 - "@cspell/dict-docker/cspell-ext.json"
 
+languageSettings:
+- languageId:
+  - markdown
+  - mdx
+  ignoreRegExpList:
+  - markdown_code_block
+  - markdown_inline_code
+
+patterns:
+- name: markdown_code_block
+  pattern: |
+    /
+        ^(\s*`{3,}).*     # match the ```
+        [\s\S]*?          # the block of code
```

Comment:
What is the difference between `[\s\S]` and `.`?

Reply:
This is the pattern provided by the cspell docs that I copied here and tested out. ChatGPT says `.` will not match newlines and `[\s\S]` is a workaround to catch multi-line code blocks. However, when I tested it locally, `.` also seemed to work for multiple lines.